frontend/app/src/components/home/P2PSwapContent.svelte

```diff
@@ -0,0 +1,295 @@
+<script lang="ts">
+    import Button from "../Button.svelte";
+    import type { MessageContext, OpenChat, P2PSwapContent } from "openchat-client";
+    import { _ } from "svelte-i18n";
+    import Clock from "svelte-material-icons/Clock.svelte";
+    import ButtonGroup from "../ButtonGroup.svelte";
+    import SwapIcon from "svelte-material-icons/SwapHorizontal.svelte";
+    import { getContext } from "svelte";
+    import { rtlStore } from "../../stores/rtl";
+    import { now500 } from "../../stores/time";
+    import SpinningToken from "../icons/SpinningToken.svelte";
+    import { toastStore } from "../../stores/toast";
+    import AreYouSure from "../AreYouSure.svelte";
+    import { i18nKey } from "../../i18n/i18n";
+    import Markdown from "./Markdown.svelte";
+    import AcceptP2PSwapModal from "./AcceptP2PSwapModal.svelte";
+
+    const client = getContext<OpenChat>("client");
+
+    export let content: P2PSwapContent;
+    export let messageContext: MessageContext;
+    export let messageId: bigint;
+    export let me: boolean;
+
+    let buttonText = "";
```

Comment:
Ideally, these dynamic text values would all be `ResourceKey | undefined` so that they can be fed into `Translatable` components in the UI.